webapp/src/js/services/db-sync.js

```diff
@@ -219,6 +228,12 @@ angular
        * @returns Promise which resolves when both directions of the replication complete.
        */
       sync: force => {
+        // Replicating once every 24 hours
+        if (!intervalPromises.allMeta) {
+          intervalPromises.allMeta = $interval(replicateToAllMeta, ALL_META_SYNC_INTERVAL);
+          replicateToAllMeta();
+        }
+
```

Comment:
This isn't ideal because this means users have to push all their data to two databases which doubles the bandwidth required. Also, if they're offline every night it'll never get pushed to the allMeta DB. Ideally webapp wouldn't ever have to know that the allMeta db exists.

I think it would be better to have a service that runs on API every 24 hours and replicates each user's meta DB into the allMeta DB one by one. That way as long as the doc makes it to CouchDB, API will take care of the rest. Does that make sense?